hack/ci/update-job-registry.py

```
@@ -0,0 +1,282 @@
+
+class Category(TypedDict):
+    name: str
+    description: str
+    jobs: list[str]
+
+
+def sparse_checkout(dest: str) -> str:
+    """Sparse-checkout only the hypershift periodic job configs from
+    openshift/release. Uses tree:0 filter so only the trees and blobs along
+    the target path are fetched."""
+    subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)